{"nodes":[{"content":"Represents the exception that is thrown when an operation on a workflow instance is not valid because the instance has completed.","nodes":[{"pos":[0,129],"content":"Represents the exception that is thrown when an operation on a workflow instance is not valid because the instance has completed.","nodes":[{"content":"Represents the exception that is thrown when an operation on a workflow instance is not valid because the instance has completed.","pos":[0,129]}]}],"pos":[1114,1244],"yaml":true},{"content":"Creates a new instance of the <xref href=\"System.Activities.WorkflowApplicationCompletedException\"></xref> class.","nodes":[{"pos":[0,113],"content":"Creates a new instance of the <ph id=\"ph1\">&lt;xref href=\"System.Activities.WorkflowApplicationCompletedException\"&gt;&lt;/xref&gt;</ph> class.","source":"Creates a new instance of the <xref href=\"System.Activities.WorkflowApplicationCompletedException\"></xref> class."}],"pos":[3540,3654],"yaml":true},{"content":"Creates a new instance of the <xref href=\"System.Activities.WorkflowApplicationCompletedException\"></xref> class with a text string description of the exception.","nodes":[{"pos":[0,161],"content":"Creates a new instance of the <ph id=\"ph1\">&lt;xref href=\"System.Activities.WorkflowApplicationCompletedException\"&gt;&lt;/xref&gt;</ph> class with a text string description of the exception.","source":"Creates a new instance of the <xref href=\"System.Activities.WorkflowApplicationCompletedException\"></xref> class with a text string description of the exception."}],"pos":[5027,5189],"yaml":true},{"content":"A description of the application completed exception.","nodes":[{"pos":[0,53],"content":"A description of the application completed exception.","nodes":[{"content":"A description of the application completed exception.","pos":[0,53]}]}],"pos":[5354,5408],"yaml":true},{"content":"Creates a new instance of the <xref href=\"System.Activities.WorkflowApplicationCompletedException\"></xref> class with the context and serialization information required to retrieve data about this exception from a serialized stream.","nodes":[{"pos":[0,232],"content":"Creates a new instance of the <ph id=\"ph1\">&lt;xref href=\"System.Activities.WorkflowApplicationCompletedException\"&gt;&lt;/xref&gt;</ph> class with the context and serialization information required to retrieve data about this exception from a serialized stream.","source":"Creates a new instance of the <xref href=\"System.Activities.WorkflowApplicationCompletedException\"></xref> class with the context and serialization information required to retrieve data about this exception from a serialized stream."}],"pos":[7014,7247],"yaml":true},{"content":"All the information required to serialize or deserialize the application completed exception object.","nodes":[{"pos":[0,100],"content":"All the information required to serialize or deserialize the application completed exception object.","nodes":[{"content":"All the information required to serialize or deserialize the application completed exception object.","pos":[0,100]}]}],"pos":[7537,7638],"yaml":true},{"content":"The source and destination of a given serialized stream and an additional caller-defined context.","nodes":[{"pos":[0,97],"content":"The source and destination of a given serialized stream and an additional caller-defined context.","nodes":[{"content":"The source and destination of a given serialized stream and an additional caller-defined context.","pos":[0,97]}]}],"pos":[7733,7831],"yaml":true},{"content":"Creates a new instance of the <xref href=\"System.Activities.WorkflowApplicationCompletedException\"></xref> class with a text string description of the exception and with an additional exception that is the cause of the application completed exception.","nodes":[{"pos":[0,251],"content":"Creates a new instance of the <ph id=\"ph1\">&lt;xref href=\"System.Activities.WorkflowApplicationCompletedException\"&gt;&lt;/xref&gt;</ph> class with a text string description of the exception and with an additional exception that is the cause of the application completed exception.","source":"Creates a new instance of the <xref href=\"System.Activities.WorkflowApplicationCompletedException\"></xref> class with a text string description of the exception and with an additional exception that is the cause of the application completed exception."}],"pos":[9197,9449],"yaml":true},{"content":"A description of the application completed exception.","nodes":[{"pos":[0,53],"content":"A description of the application completed exception.","nodes":[{"content":"A description of the application completed exception.","pos":[0,53]}]}],"pos":[9640,9694],"yaml":true},{"content":"An exception that has a causal relationship to the application completed exception. The application completed exception is thrown in response to the inner exception and can use information from the inner exception to handle the error more appropriately.","nodes":[{"pos":[0,253],"content":"An exception that has a causal relationship to the application completed exception. The application completed exception is thrown in response to the inner exception and can use information from the inner exception to handle the error more appropriately.","nodes":[{"content":"An exception that has a causal relationship to the application completed exception. The application completed exception is thrown in response to the inner exception and can use information from the inner exception to handle the error more appropriately.","pos":[0,253],"nodes":[{"content":"An exception that has a causal relationship to the application completed exception.","pos":[0,83]},{"content":"The application completed exception is thrown in response to the inner exception and can use information from the inner exception to handle the error more appropriately.","pos":[84,253]}]}]}],"pos":[9767,10021],"yaml":true},{"content":"Creates a new instance of the <xref href=\"System.Activities.WorkflowApplicationCompletedException\"></xref> class with a text string description of the exception and with a unique identifier for the exception.","nodes":[{"pos":[0,208],"content":"Creates a new instance of the <ph id=\"ph1\">&lt;xref href=\"System.Activities.WorkflowApplicationCompletedException\"&gt;&lt;/xref&gt;</ph> class with a text string description of the exception and with a unique identifier for the exception.","source":"Creates a new instance of the <xref href=\"System.Activities.WorkflowApplicationCompletedException\"></xref> class with a text string description of the exception and with a unique identifier for the exception."}],"pos":[11357,11566],"yaml":true},{"content":"A description of the application completed exception.","nodes":[{"pos":[0,53],"content":"A description of the application completed exception.","nodes":[{"content":"A description of the application completed exception.","pos":[0,53]}]}],"pos":[11748,11802],"yaml":true},{"content":"A globally unique identifier.","nodes":[{"pos":[0,29],"content":"A globally unique identifier.","nodes":[{"content":"A globally unique identifier.","pos":[0,29]}]}],"pos":[11866,11896],"yaml":true},{"content":"Creates a new instance of the <xref href=\"System.Activities.WorkflowApplicationCompletedException\"></xref> class with a text string description of the exception, a unique identifier for it, and an additional exception that is the cause of the application completed exception.","nodes":[{"pos":[0,275],"content":"Creates a new instance of the <ph id=\"ph1\">&lt;xref href=\"System.Activities.WorkflowApplicationCompletedException\"&gt;&lt;/xref&gt;</ph> class with a text string description of the exception, a unique identifier for it, and an additional exception that is the cause of the application completed exception.","source":"Creates a new instance of the <xref href=\"System.Activities.WorkflowApplicationCompletedException\"></xref> class with a text string description of the exception, a unique identifier for it, and an additional exception that is the cause of the application completed exception."}],"pos":[13316,13592],"yaml":true},{"content":"A description of the application completed exception.","nodes":[{"pos":[0,53],"content":"A description of the application completed exception.","nodes":[{"content":"A description of the application completed exception.","pos":[0,53]}]}],"pos":[13800,13854],"yaml":true},{"content":"A globally unique identifier.","nodes":[{"pos":[0,29],"content":"A globally unique identifier.","nodes":[{"content":"A globally unique identifier.","pos":[0,29]}]}],"pos":[13918,13948],"yaml":true},{"content":"An exception that has a causal relationship to the application completed exception. The application completed exception is thrown in response to the inner exception and can use information from the inner exception to handle the error more appropriately.","nodes":[{"pos":[0,253],"content":"An exception that has a causal relationship to the application completed exception. The application completed exception is thrown in response to the inner exception and can use information from the inner exception to handle the error more appropriately.","nodes":[{"content":"An exception that has a causal relationship to the application completed exception. The application completed exception is thrown in response to the inner exception and can use information from the inner exception to handle the error more appropriately.","pos":[0,253],"nodes":[{"content":"An exception that has a causal relationship to the application completed exception.","pos":[0,83]},{"content":"The application completed exception is thrown in response to the inner exception and can use information from the inner exception to handle the error more appropriately.","pos":[84,253]}]}]}],"pos":[14021,14275],"yaml":true}],"content":"### YamlMime:ManagedReference\nitems:\n- uid: System.Activities.WorkflowApplicationCompletedException\n  commentId: T:System.Activities.WorkflowApplicationCompletedException\n  id: WorkflowApplicationCompletedException\n  children:\n  - System.Activities.WorkflowApplicationCompletedException.#ctor\n  - System.Activities.WorkflowApplicationCompletedException.#ctor(System.String)\n  - System.Activities.WorkflowApplicationCompletedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  - System.Activities.WorkflowApplicationCompletedException.#ctor(System.String,System.Exception)\n  - System.Activities.WorkflowApplicationCompletedException.#ctor(System.String,System.Guid)\n  - System.Activities.WorkflowApplicationCompletedException.#ctor(System.String,System.Guid,System.Exception)\n  langs:\n  - csharp\n  name: WorkflowApplicationCompletedException\n  nameWithType: WorkflowApplicationCompletedException\n  fullName: System.Activities.WorkflowApplicationCompletedException\n  type: Class\n  assemblies:\n  - System.Activities\n  namespace: System.Activities\n  summary: Represents the exception that is thrown when an operation on a workflow instance is not valid because the instance has completed.\n  syntax:\n    content: 'public class WorkflowApplicationCompletedException : System.Activities.WorkflowApplicationException'\n  inheritance:\n  - System.Object\n  - System.Exception\n  - System.Activities.WorkflowApplicationException\n  derivedClasses:\n  - System.Activities.WorkflowApplicationTerminatedException\n  implements: []\n  inheritedMembers:\n  - System.Activities.WorkflowApplicationException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  - System.Activities.WorkflowApplicationException.InstanceId\n  - System.Exception.Data\n  - System.Exception.GetBaseException\n  - System.Exception.GetType\n  - System.Exception.HelpLink\n  - System.Exception.HResult\n  - System.Exception.InnerException\n  - System.Exception.Message\n  - System.Exception.SerializeObjectState\n  - System.Exception.Source\n  - System.Exception.StackTrace\n  - System.Exception.TargetSite\n  - System.Exception.ToString\n  - System.Object.Equals(System.Object)\n  - System.Object.Equals(System.Object,System.Object)\n  - System.Object.GetHashCode\n  - System.Object.MemberwiseClone\n  - System.Object.ReferenceEquals(System.Object,System.Object)\n  version:\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  monikers:\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.Activities/WorkflowApplicationCompletedException.xml\n  ms.technology:\n  - dotnet-standard\n  author: Erikre\n  ms.author: erikre\n  manager: erikre\n- uid: System.Activities.WorkflowApplicationCompletedException.#ctor\n  commentId: M:System.Activities.WorkflowApplicationCompletedException.#ctor\n  id: '#ctor'\n  parent: System.Activities.WorkflowApplicationCompletedException\n  langs:\n  - csharp\n  name: WorkflowApplicationCompletedException()\n  nameWithType: WorkflowApplicationCompletedException.WorkflowApplicationCompletedException()\n  fullName: WorkflowApplicationCompletedException.WorkflowApplicationCompletedException()\n  type: Constructor\n  assemblies:\n  - System.Activities\n  namespace: System.Activities\n  summary: Creates a new instance of the <xref href=\"System.Activities.WorkflowApplicationCompletedException\"></xref> class.\n  syntax:\n    content: public WorkflowApplicationCompletedException ();\n    parameters: []\n  overload: System.Activities.WorkflowApplicationCompletedException.#ctor*\n  exceptions: []\n  version:\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  monikers:\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.Activities/WorkflowApplicationCompletedException.xml\n  ms.technology:\n  - dotnet-standard\n  author: Erikre\n  ms.author: erikre\n  manager: erikre\n- uid: System.Activities.WorkflowApplicationCompletedException.#ctor(System.String)\n  commentId: M:System.Activities.WorkflowApplicationCompletedException.#ctor(System.String)\n  id: '#ctor(System.String)'\n  parent: System.Activities.WorkflowApplicationCompletedException\n  langs:\n  - csharp\n  name: WorkflowApplicationCompletedException(String)\n  nameWithType: WorkflowApplicationCompletedException.WorkflowApplicationCompletedException(String)\n  fullName: WorkflowApplicationCompletedException.WorkflowApplicationCompletedException(String)\n  type: Constructor\n  assemblies:\n  - System.Activities\n  namespace: System.Activities\n  summary: Creates a new instance of the <xref href=\"System.Activities.WorkflowApplicationCompletedException\"></xref> class with a text string description of the exception.\n  syntax:\n    content: public WorkflowApplicationCompletedException (string message);\n    parameters:\n    - id: message\n      type: System.String\n      description: A description of the application completed exception.\n  overload: System.Activities.WorkflowApplicationCompletedException.#ctor*\n  exceptions: []\n  version:\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  monikers:\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.Activities/WorkflowApplicationCompletedException.xml\n  ms.technology:\n  - dotnet-standard\n  author: Erikre\n  ms.author: erikre\n  manager: erikre\n- uid: System.Activities.WorkflowApplicationCompletedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  commentId: M:System.Activities.WorkflowApplicationCompletedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'\n  parent: System.Activities.WorkflowApplicationCompletedException\n  langs:\n  - csharp\n  name: WorkflowApplicationCompletedException(SerializationInfo, StreamingContext)\n  nameWithType: WorkflowApplicationCompletedException.WorkflowApplicationCompletedException(SerializationInfo, StreamingContext)\n  fullName: WorkflowApplicationCompletedException.WorkflowApplicationCompletedException(SerializationInfo, StreamingContext)\n  type: Constructor\n  assemblies:\n  - System.Activities\n  namespace: System.Activities\n  summary: Creates a new instance of the <xref href=\"System.Activities.WorkflowApplicationCompletedException\"></xref> class with the context and serialization information required to retrieve data about this exception from a serialized stream.\n  syntax:\n    content: protected WorkflowApplicationCompletedException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);\n    parameters:\n    - id: info\n      type: System.Runtime.Serialization.SerializationInfo\n      description: All the information required to serialize or deserialize the application completed exception object.\n    - id: context\n      type: System.Runtime.Serialization.StreamingContext\n      description: The source and destination of a given serialized stream and an additional caller-defined context.\n  overload: System.Activities.WorkflowApplicationCompletedException.#ctor*\n  exceptions: []\n  version:\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  monikers:\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.Activities/WorkflowApplicationCompletedException.xml\n  ms.technology:\n  - dotnet-standard\n  author: Erikre\n  ms.author: erikre\n  manager: erikre\n- uid: System.Activities.WorkflowApplicationCompletedException.#ctor(System.String,System.Exception)\n  commentId: M:System.Activities.WorkflowApplicationCompletedException.#ctor(System.String,System.Exception)\n  id: '#ctor(System.String,System.Exception)'\n  parent: System.Activities.WorkflowApplicationCompletedException\n  langs:\n  - csharp\n  name: WorkflowApplicationCompletedException(String, Exception)\n  nameWithType: WorkflowApplicationCompletedException.WorkflowApplicationCompletedException(String, Exception)\n  fullName: WorkflowApplicationCompletedException.WorkflowApplicationCompletedException(String, Exception)\n  type: Constructor\n  assemblies:\n  - System.Activities\n  namespace: System.Activities\n  summary: Creates a new instance of the <xref href=\"System.Activities.WorkflowApplicationCompletedException\"></xref> class with a text string description of the exception and with an additional exception that is the cause of the application completed exception.\n  syntax:\n    content: public WorkflowApplicationCompletedException (string message, Exception innerException);\n    parameters:\n    - id: message\n      type: System.String\n      description: A description of the application completed exception.\n    - id: innerException\n      type: System.Exception\n      description: An exception that has a causal relationship to the application completed exception. The application completed exception is thrown in response to the inner exception and can use information from the inner exception to handle the error more appropriately.\n  overload: System.Activities.WorkflowApplicationCompletedException.#ctor*\n  exceptions: []\n  version:\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  monikers:\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.Activities/WorkflowApplicationCompletedException.xml\n  ms.technology:\n  - dotnet-standard\n  author: Erikre\n  ms.author: erikre\n  manager: erikre\n- uid: System.Activities.WorkflowApplicationCompletedException.#ctor(System.String,System.Guid)\n  commentId: M:System.Activities.WorkflowApplicationCompletedException.#ctor(System.String,System.Guid)\n  id: '#ctor(System.String,System.Guid)'\n  parent: System.Activities.WorkflowApplicationCompletedException\n  langs:\n  - csharp\n  name: WorkflowApplicationCompletedException(String, Guid)\n  nameWithType: WorkflowApplicationCompletedException.WorkflowApplicationCompletedException(String, Guid)\n  fullName: WorkflowApplicationCompletedException.WorkflowApplicationCompletedException(String, Guid)\n  type: Constructor\n  assemblies:\n  - System.Activities\n  namespace: System.Activities\n  summary: Creates a new instance of the <xref href=\"System.Activities.WorkflowApplicationCompletedException\"></xref> class with a text string description of the exception and with a unique identifier for the exception.\n  syntax:\n    content: public WorkflowApplicationCompletedException (string message, Guid instanceId);\n    parameters:\n    - id: message\n      type: System.String\n      description: A description of the application completed exception.\n    - id: instanceId\n      type: System.Guid\n      description: A globally unique identifier.\n  overload: System.Activities.WorkflowApplicationCompletedException.#ctor*\n  exceptions: []\n  version:\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  monikers:\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.Activities/WorkflowApplicationCompletedException.xml\n  ms.technology:\n  - dotnet-standard\n  author: Erikre\n  ms.author: erikre\n  manager: erikre\n- uid: System.Activities.WorkflowApplicationCompletedException.#ctor(System.String,System.Guid,System.Exception)\n  commentId: M:System.Activities.WorkflowApplicationCompletedException.#ctor(System.String,System.Guid,System.Exception)\n  id: '#ctor(System.String,System.Guid,System.Exception)'\n  parent: System.Activities.WorkflowApplicationCompletedException\n  langs:\n  - csharp\n  name: WorkflowApplicationCompletedException(String, Guid, Exception)\n  nameWithType: WorkflowApplicationCompletedException.WorkflowApplicationCompletedException(String, Guid, Exception)\n  fullName: WorkflowApplicationCompletedException.WorkflowApplicationCompletedException(String, Guid, Exception)\n  type: Constructor\n  assemblies:\n  - System.Activities\n  namespace: System.Activities\n  summary: Creates a new instance of the <xref href=\"System.Activities.WorkflowApplicationCompletedException\"></xref> class with a text string description of the exception, a unique identifier for it, and an additional exception that is the cause of the application completed exception.\n  syntax:\n    content: public WorkflowApplicationCompletedException (string message, Guid instanceId, Exception innerException);\n    parameters:\n    - id: message\n      type: System.String\n      description: A description of the application completed exception.\n    - id: instanceId\n      type: System.Guid\n      description: A globally unique identifier.\n    - id: innerException\n      type: System.Exception\n      description: An exception that has a causal relationship to the application completed exception. The application completed exception is thrown in response to the inner exception and can use information from the inner exception to handle the error more appropriately.\n  overload: System.Activities.WorkflowApplicationCompletedException.#ctor*\n  exceptions: []\n  version:\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  monikers:\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.Activities/WorkflowApplicationCompletedException.xml\n  ms.technology:\n  - dotnet-standard\n  author: Erikre\n  ms.author: erikre\n  manager: erikre\nreferences:\n- uid: System.Activities.WorkflowApplicationException\n  parent: System.Activities\n  isExternal: false\n  name: WorkflowApplicationException\n  nameWithType: WorkflowApplicationException\n  fullName: System.Activities.WorkflowApplicationException\n- uid: System.Activities.WorkflowApplicationCompletedException.#ctor\n  parent: System.Activities.WorkflowApplicationCompletedException\n  isExternal: false\n  name: WorkflowApplicationCompletedException()\n  nameWithType: WorkflowApplicationCompletedException.WorkflowApplicationCompletedException()\n  fullName: WorkflowApplicationCompletedException.WorkflowApplicationCompletedException()\n- uid: System.Activities.WorkflowApplicationCompletedException.#ctor(System.String)\n  parent: System.Activities.WorkflowApplicationCompletedException\n  isExternal: false\n  name: WorkflowApplicationCompletedException(String)\n  nameWithType: WorkflowApplicationCompletedException.WorkflowApplicationCompletedException(String)\n  fullName: WorkflowApplicationCompletedException.WorkflowApplicationCompletedException(String)\n- uid: System.String\n  parent: System\n  isExternal: false\n  name: String\n  nameWithType: String\n  fullName: System.String\n- uid: System.Activities.WorkflowApplicationCompletedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  parent: System.Activities.WorkflowApplicationCompletedException\n  isExternal: false\n  name: WorkflowApplicationCompletedException(SerializationInfo, StreamingContext)\n  nameWithType: WorkflowApplicationCompletedException.WorkflowApplicationCompletedException(SerializationInfo, StreamingContext)\n  fullName: WorkflowApplicationCompletedException.WorkflowApplicationCompletedException(SerializationInfo, StreamingContext)\n- uid: System.Runtime.Serialization.SerializationInfo\n  parent: System.Runtime.Serialization\n  isExternal: false\n  name: SerializationInfo\n  nameWithType: SerializationInfo\n  fullName: System.Runtime.Serialization.SerializationInfo\n- uid: System.Runtime.Serialization.StreamingContext\n  parent: System.Runtime.Serialization\n  isExternal: false\n  name: StreamingContext\n  nameWithType: StreamingContext\n  fullName: System.Runtime.Serialization.StreamingContext\n- uid: System.Activities.WorkflowApplicationCompletedException.#ctor(System.String,System.Exception)\n  parent: System.Activities.WorkflowApplicationCompletedException\n  isExternal: false\n  name: WorkflowApplicationCompletedException(String, Exception)\n  nameWithType: WorkflowApplicationCompletedException.WorkflowApplicationCompletedException(String, Exception)\n  fullName: WorkflowApplicationCompletedException.WorkflowApplicationCompletedException(String, Exception)\n- uid: System.Exception\n  parent: System\n  isExternal: false\n  name: Exception\n  nameWithType: Exception\n  fullName: System.Exception\n- uid: System.Activities.WorkflowApplicationCompletedException.#ctor(System.String,System.Guid)\n  parent: System.Activities.WorkflowApplicationCompletedException\n  isExternal: false\n  name: WorkflowApplicationCompletedException(String, Guid)\n  nameWithType: WorkflowApplicationCompletedException.WorkflowApplicationCompletedException(String, Guid)\n  fullName: WorkflowApplicationCompletedException.WorkflowApplicationCompletedException(String, Guid)\n- uid: System.Guid\n  parent: System\n  isExternal: false\n  name: Guid\n  nameWithType: Guid\n  fullName: System.Guid\n- uid: System.Activities.WorkflowApplicationCompletedException.#ctor(System.String,System.Guid,System.Exception)\n  parent: System.Activities.WorkflowApplicationCompletedException\n  isExternal: false\n  name: WorkflowApplicationCompletedException(String, Guid, Exception)\n  nameWithType: WorkflowApplicationCompletedException.WorkflowApplicationCompletedException(String, Guid, Exception)\n  fullName: WorkflowApplicationCompletedException.WorkflowApplicationCompletedException(String, Guid, Exception)\n- uid: System.Activities.WorkflowApplicationCompletedException.#ctor*\n  parent: System.Activities.WorkflowApplicationCompletedException\n  isExternal: false\n  name: WorkflowApplicationCompletedException\n  nameWithType: WorkflowApplicationCompletedException.WorkflowApplicationCompletedException\n  fullName: WorkflowApplicationCompletedException.WorkflowApplicationCompletedException\n  monikers:\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.Activities/WorkflowApplicationCompletedException.xml\n- uid: System.Object.Equals(System.Object)\n  parent: System.Object\n  isExternal: false\n  name: Equals(Object)\n  nameWithType: Object.Equals(Object)\n  fullName: Object.Equals(Object)\n- uid: System.Object.Equals(System.Object,System.Object)\n  parent: System.Object\n  isExternal: false\n  name: Equals(Object, Object)\n  nameWithType: Object.Equals(Object, Object)\n  fullName: Object.Equals(Object, Object)\n- uid: System.Object.GetHashCode\n  parent: System.Object\n  isExternal: false\n  name: GetHashCode()\n  nameWithType: Object.GetHashCode()\n  fullName: Object.GetHashCode()\n- uid: System.Exception.GetType\n  parent: System.Exception\n  isExternal: false\n  name: GetType()\n  nameWithType: Exception.GetType()\n  fullName: Exception.GetType()\n- uid: System.Object.MemberwiseClone\n  parent: System.Object\n  isExternal: false\n  name: MemberwiseClone()\n  nameWithType: Object.MemberwiseClone()\n  fullName: Object.MemberwiseClone()\n- uid: System.Object.ReferenceEquals(System.Object,System.Object)\n  parent: System.Object\n  isExternal: false\n  name: ReferenceEquals(Object, Object)\n  nameWithType: Object.ReferenceEquals(Object, Object)\n  fullName: Object.ReferenceEquals(Object, Object)\n- uid: System.Exception.ToString\n  parent: System.Exception\n  isExternal: false\n  name: ToString()\n  nameWithType: Exception.ToString()\n  fullName: Exception.ToString()\n- uid: System.Exception.Data\n  parent: System.Exception\n  isExternal: false\n  name: Data\n  nameWithType: Exception.Data\n  fullName: Exception.Data\n- uid: System.Exception.GetBaseException\n  parent: System.Exception\n  isExternal: false\n  name: GetBaseException()\n  nameWithType: Exception.GetBaseException()\n  fullName: Exception.GetBaseException()\n- uid: System.Activities.WorkflowApplicationException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  parent: System.Activities.WorkflowApplicationException\n  isExternal: false\n  name: GetObjectData(SerializationInfo, StreamingContext)\n  nameWithType: WorkflowApplicationException.GetObjectData(SerializationInfo, StreamingContext)\n  fullName: WorkflowApplicationException.GetObjectData(SerializationInfo, StreamingContext)\n- uid: System.Exception.HelpLink\n  parent: System.Exception\n  isExternal: false\n  name: HelpLink\n  nameWithType: Exception.HelpLink\n  fullName: Exception.HelpLink\n- uid: System.Exception.HResult\n  parent: System.Exception\n  isExternal: false\n  name: HResult\n  nameWithType: Exception.HResult\n  fullName: Exception.HResult\n- uid: System.Exception.InnerException\n  parent: System.Exception\n  isExternal: false\n  name: InnerException\n  nameWithType: Exception.InnerException\n  fullName: Exception.InnerException\n- uid: System.Exception.Message\n  parent: System.Exception\n  isExternal: false\n  name: Message\n  nameWithType: Exception.Message\n  fullName: Exception.Message\n- uid: System.Exception.SerializeObjectState\n  parent: System.Exception\n  isExternal: false\n  name: SerializeObjectState\n  nameWithType: Exception.SerializeObjectState\n  fullName: Exception.SerializeObjectState\n- uid: System.Exception.Source\n  parent: System.Exception\n  isExternal: false\n  name: Source\n  nameWithType: Exception.Source\n  fullName: Exception.Source\n- uid: System.Exception.StackTrace\n  parent: System.Exception\n  isExternal: false\n  name: StackTrace\n  nameWithType: Exception.StackTrace\n  fullName: Exception.StackTrace\n- uid: System.Exception.TargetSite\n  parent: System.Exception\n  isExternal: false\n  name: TargetSite\n  nameWithType: Exception.TargetSite\n  fullName: Exception.TargetSite\n- uid: System.Activities.WorkflowApplicationException.InstanceId\n  parent: System.Activities.WorkflowApplicationException\n  isExternal: false\n  name: InstanceId\n  nameWithType: WorkflowApplicationException.InstanceId\n  fullName: WorkflowApplicationException.InstanceId\n"}